doc/source/whatsnew/v0.25.0.rst

```diff
@@ -99,6 +99,7 @@ Other Enhancements
 - Error message for missing required imports now includes the original import error's text (:issue:`23868`)
 - :class:`DatetimeIndex` and :class:`TimedeltaIndex` now have a ``mean`` method (:issue:`24757`)
 - :meth:`DataFrame.describe` now formats integer percentiles without decimal point (:issue:`26660`)
+- :meth:`nargsort` handles ``ExtensionArray`` with its ``_values_for_argsort`` method (:issue:`25439`)
```

Comment:
probably don't want to reference internal methods. can you reword for a pandas user.